### DISCUSSION TITLE
Cannot find community container `facerecognition`

### DISCUSSION THREAD

**Discussion by DenuxPlays** [nextcloud/all-in-one#4032] [Questions] [answered]
### Steps to reproduce
1. try to add facerecognition as a community container
2. take a look at the logs
3.

### Expected behavior 

This should work and add the container + its features.

### Actual behavior 

![image](https://github.com/nextcloud/all-in-one/assets/83671398/e91cad0f-8b0e-4f59-9226-3e69750c669d)

### Host OS 

Fedora 39 Server

#### Nextcloud AIO version 

Nextcloud AIO v7.10.0 Beta

![image](https://github.com/nextcloud/all-in-one/assets/83671398/232aeae3-7b42-493f-9812-eba3c89d467d)

#### Current channel 

Beta

#### Other valuable info 

I had the following line in my docker compose setup:
```
- AIO_COMMUNITY_CONTAINERS="facerecognition"
```
To make it work remove the `"`.
So replace it with this line:
```
- AIO_COMMUNITY_CONTAINERS=facerecognition
```

**Comment by szaimen** [comment, marked as answer]
> I had the following line in my docker compose setup:
> 
> ```
> - AIO_COMMUNITY_CONTAINERS="facerecognition"
> ```
> 
> To make it work remove the `"`. So replace it with this line:
> 
> ```
> - AIO_COMMUNITY_CONTAINERS=facerecognition
> ```
